<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlItem</class>
 <widget class="QWidget" name="ControlItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="xDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="yDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="zLabel">
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="zDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="xBaseLabel">
       <property name="text">
        <string>xBase</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="xBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="yBaseLabel">
       <property name="text">
        <string>yBase</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="yBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="zBaseLabel">
       <property name="text">
        <string>zBase</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="zBaseDoubleSpinBox">
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="minimum">
        <double>-255.000000000000000</double>
       </property>
       <property name="maximum">
        <double>255.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="select_image_button">
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="do_button">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>xDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zBaseDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
